style.css

```diff
@@ -0,0 +1,83 @@
+body {
+  margin: auto;
+  width: 375px;
+  background-color: #e5e5e5;
+}
```

Comment:
- Optional
Consider removing the fixed width from the body. What it is doing is making the body follow fixed width of 375px, so when the display size increases your app width remains same and it is causing issues as soon as display size increases 375px.
Here you can see the issue with the display size of the iPhone XR.
![image](https://user-images.githubusercontent.com/33268927/177571864-da93c39f-f833-49b4-8630-d9246f85e4ce.png)